docs/source/basics/chat_datasets.rst

```diff
@@ -0,0 +1,192 @@
+.. _chat_dataset_usage_label:
```

Comment:
Did we ever figure out how to get a nested toc tree? If not, is there a reason we're calling this section "Basics"? I would personally go with "Data and Datasets" or something like that

Reply:
once I get everything in I can make a pass and organize everything better, perhaps with @svekars 's help